src/test/resources/main_test-expected_output.json

```diff
@@ -0,0 +1 @@
+{"total":15,"404":{"urls":[],"size":0},"dead":0,"50X":{"urls":[],"size":0},"url":"http://home.mcom.com/home/welcome.html"}
```

Comment:
@vladzasoba please make sure to format the json as requested and also use only underscores